### DISCUSSION TITLE
Plan to support CIS v1.4.0

### DISCUSSION THREAD

**Discussion by lin-ye-66** [prowler-cloud/prowler#1334] [General]
Prowler supports CIS v1.2.0 currently. Is there a plan to support v1.4.0?

**Comment by n4ch04** [comment]
Hi @lin-ye-66 we support right now CIS v1.3, next version will be included when we release Prowler version 3.0

**Reply by lin-ye-66** [reply]
Thank you for your response! That's great, and looking forward to version 3.0!

**Reply by toniblyx** [reply]
CIS 1.4 and 1.5 for AWS is already implemented in Prowler v3. Enjoy it!
More info at https://docs.prowler.cloud/en/latest/tutorials/compliance/

**Comment by toniblyx** [comment]
CIS 1.4 and 1.5 for AWS is already implemented in Prowler v3. Enjoy it!
More info at https://docs.prowler.cloud/en/latest/tutorials/compliance/
